src/classlibnative/float/Float.nativeproj

```diff
@@ -28,7 +28,7 @@
   </PropertyGroup>
   <!--Leaf Project Items-->
   <ItemGroup>
-    <CppCompile Include="FloatNative.cpp" />
+    <CppCompile Include="FloatDouble.cpp" />
```

Comment:
I believe (in the case of *.*proj) casing differences are being handled already. However, I have adjusted the casing to match what exists on disk.